src/openrct2/core/Numerics.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
```

Comment:
Missing copyright header. When copying, remember it's 2019

Reply:
Sure. Will add it to my IDE's template while I'm at it. 
Are we going to update the year numbering every now and then? 
I feel like that's horrible for version control and CI.